legacy/src/main/java/org/opensearch/sql/legacy/plugin/RestSqlAction.java

```diff
@@ -245,7 +276,7 @@ private static boolean isExplainRequest(final RestRequest request) {
         return request.path().endsWith("/_explain");
     }
 
-    private static boolean isClientError(Exception e) {
+    public static boolean isClientError(Exception e) {
```

Comment:
I may miss it but who requires this public accessibility?

Reply:
Sharp eyes! It was left from previous iterations. Fixed in 4412631d, thank you.